test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -1768,8 +1768,8 @@ TEST_P(ExtProcIntegrationTest, PerRouteGrpcService) {
 TEST_P(ExtProcIntegrationTest, RequestAndResponseMessageNewTimeoutWithHeaderMutation) {
   // Set envoy filter timeout to be 200ms.
   proto_config_.mutable_message_timeout()->set_nanos(200000000);
-  // Config max_message_timeout proto to 10s to enable the new timeout API.
-  proto_config_.mutable_max_message_timeout()->set_seconds(10);
+  // Config max_message_timeout proto to 60s to enable the new timeout API.
+  proto_config_.mutable_max_message_timeout()->set_seconds(60);
```

Comment:
Please use the TSAN_TIMEOUT_FACTOR instead of hardcoding higher timeout for all test types, if this timeout is only affecting sanitizer builds.

Reply:
Sure, just changed to use TestUtility::DefaultTimeout in all these cases.